parameter/ParameterMgr.cpp

```diff
@@ -592,8 +587,8 @@ bool CParameterMgr::loadSettings(string& strError)
 
     if (!success && !_bFailOnFailedSettingsLoad) {
         // Load can not fail, ie continue but log the load errors
-        log_info(strLoadError);
-        log_info("Failed to load settings, continue without domains.");
+        _logger.info() << strLoadError;
+        _logger.info() << "Failed to load settings, continue without domains.";
```

Comment:
_logger.info() and _logger.warning() could be wrapped in a private method in ParameterMgr